File: SportHub_client/lib/entities/user_info.dart
class UserInfo {
  String firstName; //
  String lastName; //
  String country; //
  String city; //
  String dateOfBirth; //
  String sportLevel; //
  double height; //
  double weight; //
  String aboutMe; //
  String motivation; //
  String avatarUrl;
  String userId;

  UserInfo(
      {this.firstName,
      this.lastName,
      this.country,
      this.city,
      this.dateOfBirth,
      this.sportLevel,
      this.height,
      this.weight,
      this.aboutMe,
      this.motivation,
      this.avatarUrl,
      this.userId});

  UserInfo.fromJson(Map<String, dynamic> json) {
    firstName = json['firstName'];
    lastName = json['lastName'];
    country = json['country'];
    city = json['city'];
    dateOfBirth = json['dateOfBirth'];
    sportLevel = json['sportLevel'];
    height = json['height'];
    weight = json['weight'];
    aboutMe = json['aboutMe'];
    motivation = json['motivation'];
    avatarUrl = json['avatarUrl'];
    userId = json['userId'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['firstName'] = this.firstName;
    data['lastName'] = this.lastName;
    data['country'] = this.country;
    data['city'] = this.city;
    data['dateOfBirth'] = this.dateOfBirth;
    data['sportLevel'] = this.sportLevel;
    data['height'] = this.height;
    data['weight'] = this.weight;
    data['aboutMe'] = this.aboutMe;
    data['motivation'] = this.motivation;
    data['avatarUrl'] = this.avatarUrl;
    data['userId'] = this.userId;
    return data;
  }
}

File: SportHubserver/SportHub.Core/Entities/UserInfo.cs
using SportHub.Core.Entities.Base;
using System;
using System.Collections.Generic;
using System.Text;

namespace SportHub.Core.Entities
{
    public class UserInfo : Entity
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Country { get; set; }
        public string City { get; set; }
        public string DateOfBirth { get; set; }
        public string SportLevel { get; set; }
        public float Height { get; set; }
        public float Weight { get; set; }
        public string AboutMe { get; set; }
        public string Motivation { get; set; }
        public string AvatarUrl { get; set; }

        public Guid UserId { get; set; }
        public User User { get; set; }

        public UserInfo(string firstName, string lastName, string country, string city, string dateOfBirth, 
            string sportLevel, float height, float weight, string aboutMe, string motivation, string avatarUrl, Guid userId)
        {
            FirstName = firstName;
            LastName = lastName;
            Country = country;
            City = city;
            DateOfBirth = dateOfBirth;
            SportLevel = sportLevel;
            Height = height;
            Weight = weight;
            AboutMe = aboutMe;
            Motivation = motivation;
            AvatarUrl = avatarUrl;
            UserId = userId;
        }
    }
}
